packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js

```diff
@@ -50,18 +90,37 @@ export default class OvhManagerNetAppNetworkConfigurationCtrl {
     this.trackClick('confirm');
 
     this.NetappNetworkConfigurationService.linkVrackServiceToEfs(
+      this.selectedVrack.internalName,
       this.selectedVrackService,
       this.selectedSubnet,
       this.storage,
     )
-      .then(() => {
+      .then((data) => {
         this.trackSuccess();
         this.goBack().then(() => {
           this.Alerter.success(
             this.$translate.instant(
               'netapp_network_configuration_success_message',
             ),
           );
+
+          if (
+            data.vrackAssociationStatus &&
+            !data.vrackAssociationStatus.success
```

Comment:
so if `data.vrackAssociationStatus` is null no need to display any alert ?

Reply:
I would say yes but I am still not sure :)